### ISSUE TITLE
incorrectly shows the number of records in the table PostgreSQL 

### ISSUE THREAD

**Issue by OmlineEditor** [adminerevo/adminerevo#112]
I have a table with 3 records, but when viewing the general view, I can't see the real values of how many records I have in the table.

![client](https://github.com/adminerevo/adminerevo/assets/13348933/d8a13d9a-7b1e-4128-a91d-4c3ce27a23cb)

![client2](https://github.com/adminerevo/adminerevo/assets/13348933/92e360d5-57a5-4ce3-9b91-3894f93d9ace)

**Comment by mckaygerhard** [comment]
yeah i can confirm this happened but the symbol `~` means "approximately" or "more or less" i see that happened in both mysql and postgresql.. but only few times.. 

i guess this is cos calculations over web need cached the result etc etc.. so i guess it needs a new feature like "compute" in the table page!

this is more a feature like an issue i guess i dont know..
